WooCommerce/Classes/ViewRelated/Dashboard/Settings/Beta features/BetaFeaturesViewController.swift

```diff
@@ -120,8 +121,8 @@ private extension BetaFeaturesViewController {
     func configureProductsSwitch(cell: SwitchTableViewCell) {
         configureCommonStylesForSwitchCell(cell)
 
-        let title = NSLocalizedString("Product editing",
-                                      comment: "My Store > Settings > Experimental features > Product editing")
+        let title = NSLocalizedString("Products",
```

Comment:
Should it be `Add Products`, to be more clear with the users about what they are enabling? What do you think?

Reply:
(responded in https://github.com/woocommerce/woocommerce-ios/pull/3008#issuecomment-712530545)